### PR TITLE
Popup

### PR DESCRIPTION
**Pull request by NduatiKagiri**
In this branch we:

- Designed and styled the popup
- Added dynamics on the Works section
- Added popup functions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,123 +65,47 @@ <h2>My Recent Works</h2>
...
+                  </div>
+                  <div class="project-links">
+                    <a href="https://nduatikagiri.co.ke" id="project-link-1">See Live &nbsp; &nbsp; &nbsp; <i class="fas fa-external-link" aria-hidden="true"></i></a>
+                    <a href="https://github.com/NduatiKagiri/my_portfolio" id="project-link-2">See Source &nbsp; &nbsp; &nbsp; <i class="fab fa-github" aria-hidden="true"></i></a>
+                  </div>
+                  <div class="project-buttons">
+                    <button type="button" name="button"><i class="fal fa-long-arrow-left" aria-hidden="true"></i> previous project</button>
+                    <button type="button" name="button">next project  <i class="fal fa-long-arrow-right" aria-hidden="true"></i></button>
+                  </div>
+                </div>
```
Comment (Omar-Muhamad) [inline index.html:103]: - [ ] Kindly the project pop-up card modal suppose to be generated dynamically using javascript too like the project work cards so Please remove this code from HTML and implement it using JS.

File: assets/js/works.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  {
+    heading: 'Professional Art Printing Data More',
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    languages: [
+      'html',
+      'bootstrap',
+      'ruby',
+    ],
+  },
+];
```
Comment (Omar-Muhamad) [inline assets/js/works.js:29]: - [ ] The site suppose to have 1 muti post project and 6 other project cards like the Figma design you just implement 3 projects only please add the other projects too and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image |

File: assets/js/works.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      <div class="work-card-tools">
+        <ul>
+          ${langs}
+        </ul>
+      </div>
+      <div class="work-card-button">
+        <button id="${index}">See Project</button>
+      </div>
+    </div>
+    `;
```
Comment (Omar-Muhamad) [inline assets/js/works.js:55]: - [ ] For the pop-up work card design in the desktop version you forgot to add the project title on the top of the card and the project description suppose to be beside the project image not below it, So Please fix that and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  |image |

- [ ] In the mobile version of the pop-up card please fix the design to be responsive and prevent side-scrolling and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | chrome-capture-2022-4-12 |

**Review by Omar-Muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: assets/js/works.js (hunk 3 above)
Comment (danmainah) [reply assets/js/works.js:143]: - [ ] @Team, great work implementing the changes requested, there is still some differences in comparison to the figma design check the comparison below:
Desktop


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

In mobile design kindly remove the extra space below the `see Iive` and `see code` buttons.Fix the design to be responsive and prevent side-scrolling

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image|  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Team, 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Github flow was correctly used


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁 
# Hello  @NduatiKagiri 😃 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️
In desktop version, the image is overflowing from the popup window.
Screen Shot 2022-05-12 at 9 54 29 PM

2. As asked by the previous code reviewer, Kindly remove the extra spaces in the button and side-scrolling in the mobile version to make it responsive.
Screen Shot 2022-05-12 at 9 57 24 PM


As you may have lost all review attempts but you can still use extra reviews. Kindly fix the issues and ask for a review again. 👍🏻 😄 


_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**